docs/release-notes/highlights-1.0.0-beta1.asciidoc

```diff
@@ -1,20 +1,47 @@
 
-TODO
+ECK now uses StatefulSets under the hood to orchestrate Elasticsearch clusters. Prior to 1.0.0-beta1, upgrading a cluster storing large volumes of data could take hours or even days because data had to be replicated to newly created Elasticsearch nodes. With StatefulSets, you can simply reuse persistent volumes and significantly reduce the time and resources used to upgrade an Elasticsearch cluster. See <<{p}-orchestration>> for more detail.
 
 [float]
-=== TODO
+[id="{p}-release-v1beta1"]
+=== Resource version promotion
 
-TODO
+Custom resources have graduated to version `v1beta1`.  Old resources must be migrated to the new version to work with ECK 1.0.0-beta1. Resources created using an older version will continue to run but the operator will not manage them. To receive all the benefits of ECK 1.0.0-beta1, we recommend deleting old custom resources and recreating them. See <<{p}-upgrading-eck>> for more information.
 
 [float]
-=== Breaking Changes
+[id="{p}-release-validation"]
+=== Validation changes
+
+OpenAPI validation of resources on admission is no longer enforced on Kubernetes versions 1.13 or lower. This is due to a limitation in older versions of Kubernetes where multiple versions for CRDs are not supported.
+
+Webhook validation has also temporarily been removed from this release. It is currently being reworked.
+
+[float]
+[id="{p}-release-tls"]
+=== More control over TLS settings
+
+Users have more control over TLS settings of the Elasticsearch HTTP layer, Kibana, and APM Server in this release. It is now possible to use external certificates from well-known CAs, utilities like cert-manager, or even completely disable automatic provisioning of TLS certificates. The latter can be useful when TLS is handled at a different layer, for instance, with Istio deployments. See: <<{p}-setting-up-your-own-certificate>> and <<{p}-disable-tls>>. 
+
+[float]
+[id="{p}-release-secure-settings"]
+=== Secure settings enhancements
+
+Secure settings have received multiple ergonomics improvements. It is now possible to specify secure settings from multiple secrets. It is also possible to specify specific items in a Kubernetes secret to be used as a secure setting, rather than having to use all items in a secret.
+
+[float]
+[id="{p}-release-default-limits"]
+=== Default limits
 
-TODO
+Default limits are now set for the various resource types if none are explicitly specified. Previously, if left at the default, Pods would consume unbounded amounts of memory, which could lead them to be OOM killed. This allows for a better experience in demo scenarios. We recommend in production that both requests and limits be set appropriately. See <<{p}-managing-compute-resources>> for more information.
+
+[float]
+[id="{p}-breaking-changes-1.0.0-beta1"]
+=== Breaking Changes
 
 Attempting to delete resources created with a 0.9.0 version will hang if ECK 1.0.0-beta1 is running. To unblock the deletion, remove any registered finalizer from the resource:
 
```

Comment:
feels somehwhat out of place to have kubectl commands in the release highlights doc

Reply:
That's fair, I'll move the actual steps to the upgrading doc and link to it